src/aggregate/aggregate_request.c

```diff
@@ -1248,46 +1237,72 @@ static void PushUpStream(ResultProcessor *rp_to_place, ResultProcessor *rp) {
 // The Buffer-Locker rp is added as the upstream of the first result processor that might
 // access Redis keyspace.
 // The Unlocker is places so that its upstream rp will be the last to access Redis keyspace.
-static void SafeRedisKeyspaceAccessPipeline(AREQ *req, QueryError *status) {
+static void SafeRedisKeyspaceAccessPipeline(AREQ *req) {

```

Comment:
please add in one of the comments that we assume the `rootProc` does not access redis itself (for future proofing)